annotators/emotion_detection/test.py

```diff
@@ -0,0 +1,12 @@
+# import requests
+
+
+# def test():
+#     response = requests.post("http://0.0.0.0:8026/model", json={"personality": ["Hello world"]})
+#     assert response.status_code == 200
+#     assert response.json() == [{"EXT": 0, "NEU": 1, "AGR": 1, "CON": 0, "OPN": 1}]
+#     print("SUCCESS")
```

Comment:
why tests are not added?

Reply:
They are copied from `personality_detection`. Removed